Assets/StreamingAssets/Data/Furniture.xml

```diff
@@ -277,6 +277,45 @@
     <LocalizationCode>furn_metal_smelter</LocalizationCode>
     <UnlocalizedDescription>furn_metal_smelter_desc</UnlocalizedDescription>
   </Furniture>
+	
+<Furniture objectType="Metal Furnace">
+    <Name>Metal Furnace</Name>
+    <Description>Smelts your raw ores into metal.</Description>
+    <TypeTag>Processor</TypeTag>
+    <MovementCost>1</MovementCost>
+    <PathfindingModifier>1.5</PathfindingModifier>
+    <PathfindingWeight>2</PathfindingWeight>
+    <Width>3</Width>
+    <Height>3</Height>
+
+    <BuildingJob jobTime="0" />
+
+    <JobSpotOffset X="1" Y="0" />
+
+	<FactoryInfo>
+		<PossibleProductions>
+			<ProductionChain name="Iron smelting" processingTime="4">
+				<Input>
+					<Item objectType="Raw Iron" amount="3" slotPosX="0" slotPosY="0" />
```

Comment:
example: you have one machine that creates some chips (or something), it has 2 production chains defined:
- crappy chip (requires 2 inputs - copper wire 1x, silicon wafer 1x)
- super cool chip (requires 3 inputs - copper wire 1x, silicon wafer 2x, diamond 1x)
Not sure I get the conversion ratios.. it's explicitly set by amounts on input and output (factorio style)